crates/aptos-telemetry-service/src/types/telemetry.rs

```diff
@@ -29,3 +31,23 @@ pub(crate) struct BigQueryRow {
     pub event_timestamp: u64,
     pub event_params: Vec<serde_json::Value>,
 }
+
+#[derive(Debug, Deserialize, Clone)]
+pub struct RemoteNodeConfigEntry {
+    pub node_type: NodeType,
+    pub chain_id: ChainId,
+    pub peer_ids: Vec<PeerId>,
+    pub node_config: serde_yaml::Value,
+}
+
+#[derive(Debug, Deserialize, Clone)]
+pub struct RemoteNodeConfigsFile {
+    pub version: u64,
+    pub node_configs: Vec<RemoteNodeConfigEntry>,
```

Comment:
Shouldn't the node configs file contain a list of remote config updates? E.g., if rolling out to nodes in three stages, won't you want to append in case some validators didn't get the update yet, or wiped their disks? Maybe it's overkill.